var/spack/repos/builtin/packages/byacc/package.py

```diff
@@ -13,10 +13,11 @@ class Byacc(AutotoolsPackage):
     documentation. Specifications that take advantage of undocumented
     features of AT&T Yacc will probably be rejected."""
 
-    homepage = "https://github.com/grandseiken/byacc"
-    git      = "https://github.com/grandseiken/byacc.git"
+    homepage = "https://invisible-island.net/byacc/byacc.html"
+    url      = "https://invisible-island.net/datafiles/release/byacc-20210808.tar.gz"
```

Comment:
Does this url exist?

Reply:
Looks like it no longer works. Changed to FTP